### DISCUSSION TITLE
Improve transactions color scheme to avoid confusion

### DISCUSSION THREAD

**Discussion by MaudSimon** [alephium/alephium-frontend#378] [Feature requests]
Several users reported that the red color for outgoing transactions was confusing as at first glance it looks like a failed transaction from the color scheme. 

One suggestion was made to use the same color for both with just up and down arrow. 
(Personally I think that having a slightly different color is helpful, just not red so it's not confusing).

<img width="616" alt="image" src="https://github.com/alephium/alephium-frontend/assets/52083111/2bbe6cd0-040a-4f8a-87c5-3c736fc5c943">

See latest report: https://discord.com/channels/747741246667227157/948144672402972682/1209953851784953917

**Comment by rabTAI** [comment]
VendingMachine deployed transactions (last two)

**Comment by nop33** [comment]
What about the colors we have in the mobile wallet? They are pretty distinct

![image](https://github.com/alephium/alephium-frontend/assets/1579899/1a20f48e-c34f-4e90-ae32-a68a9845a00f)

**Comment by nop33** [comment]
Implemented in https://github.com/alephium/alephium-frontend/pull/1690. Closing
